fixtures/module/outputs.tf

```diff
@@ -1,15 +1,30 @@
-output "env_name" {
-    value = "${module.test_module.env_name}"
+output "env_name_1" {
```

Comment:
I don't see any renames in the tests to account for `env_name` -> `env_name_1`. Are these outputs used anywhere?